PWG/EMCAL/config/AliEmcalCorrectionConfiguration.yaml

```diff
@@ -63,6 +63,13 @@ CellEnergy:                                         # Cell Energy correction com
     customRecalibFilePath: ""                       # Full path including .root file for custom recalibration object
     cellsNames:                                     # Names of the cells input objects which should be attached to the correction
         - defaultCells                              # This object is defined above in the cells section of the input objects
+CellSingleChannelCalibration:                       # Cell Single Channel Energy correction component
+    enabled: false                                  # Whether to enable the task
+    createHistos: false                             # Whether the task should create output histograms
+    enableNewTempCalib: false                       # Whether the new temperature calibration parmeters should be used (available for Run1 and Run2)
```

Comment:
Is this option used / needed? I don't see any other references to it

Reply:
CellSingleChannelCalibration is the correction component for the Single Channel calibration. Locally, I have added and enabled it in  PWG/EMCAL/config/EMCalSampleConfig.yaml for testing. If you are referring to enableNewTempCalib, then yes, that should be removed. And yes, I think most of the commit can be squashed. I will do that.

Reply:
Yeah, I was just referring to the `enableNewTempCalib` option. The comment ended up highlighting more than I expected. Thanks!